aten/src/ATen/native/mps/operations/Indexing.h

```diff
@@ -0,0 +1,56 @@
+//  Copyright © 2022 Apple Inc.
+
+#include <ATen/ATen.h>
+#include <ATen/Tensor.h>
+#include <ATen/Utils.h>
+#include <ATen/mps/MPSStream.h>
+#include <ATen/native/mps/TensorFactory.h>
+#include <c10/core/ScalarType.h>
+#include <torch/library.h>
+#include <unordered_map>
+
+#ifdef __OBJC__
+#include <MetalPerformanceShaders/MetalPerformanceShaders.h>
+#endif
```

Comment:
Aren't those already included by `MPSStream.h`? (And at the very least, `#ifdef __OBJC__` is superfluous in `.mm` file, isn't it?  
```suggestion
```

Reply:
Yea, that  `__OBJC__`  check doesn't make sense to be here - will update

Reply:
Done.